components/03-modules/gallery/_gallery.scss

```diff
@@ -0,0 +1,102 @@
+$gallery__stage-margin-bottom      : $spacer--medium !default;
+$gallery__nav-vertical-margin-right: 40px !default;
+$gallery__thumb-spacing            : $spacer--medium !default;
+$gallery__thumb-border             : none !default;
+$gallery__thumb-border--active     : 1px solid $green !default;
+$gallery__thumb-width              : 48px !default;
+$gallery__thumb-width--large       : 80px !default;
+$gallery__thumb-arrow-width        : 48px !default;
+$gallery__thumb-arrow-height       : 48px !default;
+$gallery__icon-arrow-width         : 12px !default;
+
+.gallery {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: flex-start;
+    width: 100%;
+    margin: 0;
+    padding: 0;
+    list-style-type: none;
+    flex-direction: column;
+    &--vertical {
+        @include mq($screen-m) {
+            flex-direction: row;
+        }
+    }
+    &__frame {
+        position: relative;
+    }
+    &__stage {
+        order: 1;
+        margin-bottom: $gallery__stage-margin-bottom;
```

Comment:
in future -> if you have only margin like this `$spacer` or simillar without any other values and it doesn't repeat in file don't create specific variables

Reply:
so we would need refactor a lot of code as I see this approach in many places.

Reply:
as we discussed on chat, not necessary anymore